core/wire/session.go

```diff
@@ -184,22 +184,23 @@ func (s *Session) handshake() error {
 	defer handshake.Reset()
 	var (
 		prologueLen = 1
+		keyLen      = 32
```

Comment:
Which `key` is this the length of? Is there a constant in `nyquist` we should be referencing here?

Reply:
ah yeah Nyquest has a constant for that... i will fix